scripts/magpietts/infer_and_evaluate.py

```diff
@@ -240,7 +244,7 @@ def run_inference(
     else:
         exp_name = ""
 
-    checkpoint_name = checkpoint_file.split("/")[-1].split(".ckpt")[0]
+    checkpoint_name = checkpoint_file.split("/")[-1].split(".ckpt")[0] if checkpoint_file is not None else checkpoint_name
```

Comment:
Can you remove this line? checkpoint_name is defined in the if else above, I don't know why this returned.